examples/src/main/kotlin/usageExamples/update/Update.kt

```diff
@@ -0,0 +1,59 @@
+package usageExamples.update
+
+// :replace-start: {
+//    "terms": {
+//       "CONNECTION_URI_PLACEHOLDER": "\"<connection string uri>\"",
+//       "import io.github.cdimascio.dotenv.dotenv\n": ""
+//    }
+// }
+// :snippet-start: update-usage-example
+import com.mongodb.MongoException
+import com.mongodb.client.model.Filters
+import com.mongodb.client.model.UpdateOptions
+import com.mongodb.client.model.Updates
+import com.mongodb.kotlin.client.coroutine.MongoClient
+import io.github.cdimascio.dotenv.dotenv
+import kotlinx.coroutines.runBlocking
+import java.util.*
```

Comment:
Nit: Star import.  

Use `import java.time.LocalDateTime` instead of `java.util.Date`

Reply:
Updated. (And updated my IntelliJ Imports settings 😅). 

I've also cut tickets to change any instance of Date class to LocalDateTime and to update any star imports in the docs already published.